bool isTakeoff
geometry_msgs/PoseStamped poseStamped
--- 